flink-connector-aws/flink-connector-dynamodb/src/main/java/org/apache/flink/connector/dynamodb/table/RowDataToAttributeValueConverter.java

```diff
@@ -43,14 +47,35 @@ public class RowDataToAttributeValueConverter {
 
     private final DataType physicalDataType;
     private final TableSchema<RowData> tableSchema;
+    private final Set<String> primaryKeys;
 
-    public RowDataToAttributeValueConverter(DataType physicalDataType) {
+    public RowDataToAttributeValueConverter(DataType physicalDataType, Set<String> primaryKeys) {
```

Comment:
nit: could we overload the constructor instead and pass empty set on default?

Reply:
Instead of duplicating logic, it will be better to call constructor with all parameters from one with defaults. This way there only 1 place to change if required:
```
    public RowDataToAttributeValueConverter(DataType physicalDataType) {
        this(physicalDataType, Collections.emptySet());
    }
```